package.json

```diff
@@ -16,6 +16,7 @@
     "eslint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
     "prettier": "prettier --check '**/*'",
     "test": "jest",
+    "jest:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --no-cache",
```

Comment:
This is just something helpful I added when trying to debug a jest test using the Chrome debugger